umbraco-cloud/release-notes/2023-04-releasenotes.md

```diff
@@ -0,0 +1,27 @@
+# Release Notes, April 2023
+
+## Key Takeaways
+
+* **Reset Authenticator App settings** - It is now a easy task for your cloud organization administrator to reset your authenticator app settings if you start using a new mobile or want to switch to another authenticator app.
+* **Other Umbraco Cloud updates** - Umbraco Cloud has been updated with a few other features and updates worth mentioning.
+
+## Reset Authenticator App settings
+
+The latest addition to the multi-factor-authentication feature is the option for **administrators of an organization to reset the authenticator app settings**. This can be done for every member of the organization that is using an authenticator app during the login process to Umbraco Clou.
```

Comment:
```suggestion
The latest addition to the multi-factor-authentication feature is the option for **administrators of an organization to reset the authenticator app settings**. This can be done for every member of the organization that is using an authenticator app during the login process to Umbraco Cloud.
```